util/proto/marshal_test.go

```diff
@@ -791,3 +802,29 @@ func testFixed64Marshal(t *testing.T, n uint64, wrongField bool) {
 		require.EqualValues(t, 0, result.FieldI)
 	}
 }
+
+func testFixed32Marshal(t *testing.T, n uint32, wrongField bool) {
+	var (
+		custom    = stablePrimitives{FieldK: n}
+		transport = test.Primitives{FieldK: n}
+	)
+
+	result := testMarshal(t, custom, transport, wrongField)
+
+	if !wrongField {
+		require.Equal(t, n, result.FieldK)
+	} else {
+		require.EqualValues(t, 0, result.FieldK)
+	}
+}
+
+func TestFixed32Marshal(t *testing.T) {
```

Comment:
It will look better on top together with other `Test*Marshal()` functions, in my opinion.

Reply:
Moved higher